templates/admin/layout.html.twig

```diff
@@ -74,8 +74,12 @@
 {% block footer %}
     <footer class="bottom">
         <ul class="bottom-legal">
-            <li>Helpless Kiwi 0.0.1</li>
-            {% if app.environment == 'dev' %}<li>Development mode</li>{% endif %}
+            <li>Helpless Kiwi</li>
+            {% if app.environment == 'dev' %}
+                <li>Development mode</li>
+            {% elseif version != '' %}
+                <li>versie {{ version }}</li>
```

Comment:
This entire bar of text follows a certain pattern (capitalized English text). lets keep that here as well. I also assume the `INSTALLED_VERSION` just gives the date (`xxxx-xx-xx`), and not the whole `release xxxx-xx-xx`.

Besides that, good change!

(perhaps change line 80 to `version = 'prod'`. not sure if there can be a scenario where this matters tho...

Reply:
There actually is a scenario where this matters: manual installations! Only installations by the updater script will contain the environment variable.

You're correct in your assumption that INSTALLED_VERSION is just the date.